_posts/2019-10-27-Merging-Multiple-SimpleCov-coverage-results.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Merging-Multiple-SimpleCov-coverage-results
+date: 2019-10-29 13:56:00
+categories: ["Rails", "Simplecov", "Upgrades"]
+author: bronzdoc
+---
+
+As part of our Roadmap service at FastRuby.io, we have to analyze the test suite of the application we are upgrading to give a proper estimate on how long it will take us to upgrade. We use SimpleCov for this.
```

Comment:
@bronzdoc Please break this long sentence into two: `we have to analyze the test suite of the application we are upgrading to give a proper estimate on how long it will take us to upgrade`

Reply:
@bronzdoc Please add links for Roadmap, FastRuby and SimpleCov